tests/foreman/api/test_lifecycleenvironment.py

```diff
@@ -10,7 +10,7 @@
 
 :CaseLevel: Acceptance
```

Comment:
The same comment as I left for `CaseLevel` in `tests/foreman/api/test_activationkey.py` - it probably makes more sense to change it to `Component`.